src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -570,7 +572,7 @@ static unique_ptr<Picture> _imageBuildHelper(SvgLoaderData& loaderData, SvgNode*
         if (!_isValidImageMimeTypeAndEncoding(&href, &mimetype, &encoding)) return nullptr; //not allowed mime type or encoding
         char *decoded = nullptr;
         if (encoding == imageMimeTypeEncoding::base64) {
-            auto size = b64Decode(href, strlen(href), &decoded);
+            auto size = b64Decode(href, strlen(href), &decoded, &b64DecodeSkipXML);
```

Comment:
I think only `_skipXmlEntities()` is necessary for custom decoding. Should it have an entirely duplicated decoding logic in svgLoader?